README.md

```diff
@@ -77,6 +77,44 @@ const response = await firestoreService.get('regions/32');
 
 The response will have the information about the region with id 32.
 
+#### Tools to make queries
+
+Firestore service allows you to use certain tools to manipulate your data in the query. For this you should add a body into the request.
```

Comment:
it's not a "body into the request" is an extra param in the function call, GET methods don't use the body.